.github/workflows/deploy-application.yml

```diff
@@ -86,4 +86,4 @@ jobs:
           cf_password: ${{ secrets.CF_PASSWORD }}
           cf_org: gsa-tts-oros-fac
           cf_space: ${{ env.space }}
-          command: cf run-task gsa-fac -k 6G -m 1G --name load_data --command ./load_data.sh
+          command: chmod +x ./load_data.sh && cf run-task gsa-fac -k 6G -m 1G --name load_data --command ./load_data.sh
```

Comment:
This won't make a difference because the `chmod` will happen to the `load_data.sh` file in the GHA runner on the local checkout, while the `run-task` will happen in the context of a new container.

Reply:
Good catch, that needed to be in `--command " "`